src/current/v25.2/detect-hotspots.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Detect Hotspots
+summary: Learn how to detect hotspots using real-time monitoring and historical logs in CockroachDB.
+toc: true
+---
+
+This page provides practical guidance on identifying common [hotspots]({% link {{ page.version.version }}/understand-hotspots.md %}) in CockroachDB clusters using real-time monitoring and historical logs. This tutorial assumes that you have identified a metrics outlier in your cluster. It focuses on CPU and latch contention metrics to help you identify hot-key and hot-index scenarios.
```

Comment:
"hot-key" and "hot-index" do not need to be hyphenated

"hot key" could link to the relevant section below (as could "hot index")

do we have anything to link to for "identified a metrics outlier"? i did a search of our docs for "metrics outlier" and got [these results](https://www.cockroachlabs.com/docs/search.html?query=metrics+outlier), some of which look promising - maybe https://www.cockroachlabs.com/docs/cockroachcloud/metrics-overview.html ?